app/models/scheduled_message.rb

```diff
@@ -2,4 +2,18 @@ class ScheduledMessage < ActiveRecord::Base
   validates :body, presence: true
   validates :days_after_start, presence: true
   validates :title, presence: true
+
+  before_save :escape_specific_characters
+
+  def body
+    self[:body].html_safe if self[:body]
+  end
+
+  protected
+
+  def escape_specific_characters
```

Comment:
oops, I was totally wrong about that. We are using it in the callback above (which I am suggesting we get rid of and instead make this a public method that we use in the `MessageSender` service class...thoughts?)